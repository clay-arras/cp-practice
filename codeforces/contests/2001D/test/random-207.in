1
8
11 10 24 12 13 8 20 13
