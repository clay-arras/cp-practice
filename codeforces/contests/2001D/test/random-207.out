7
11 10 24 12 13 8 20
